reference/v8.json

```diff
@@ -158,17 +158,17 @@
     "maxzoom": {
       "type": "number",
       "default": 14,
-      "doc": "Maximum zoom to preserve detail at."
+      "doc": "Maximum zoom level at which to create vector tiles (higher means greater detail at high zoom levels but slower performance)."
     },
     "buffer": {
       "type": "number",
       "default": 64,
-      "doc": "Tile buffer on each side."
+      "doc": "Tile buffer size (higher means fewer rendering artifacts near tile edges but slower performance)."
```

Comment:
"on each side" may be an important clarification since it could also be read as "total buffer in one dimension".